### ISSUE TITLE
Where is the user guide please?

### ISSUE THREAD

**Issue by murugappanrm** [tattersoftware/codeigniter4-reports#17]
Hi,

How do you publish a CI tool/component and expect everyone to guess the User Guide?

**Comment by MGatner** [comment]
All docs are in the README.

This package is very old and in need of a lot of updates, one of those definitely being better documentation. It's due for updates pretty soon because of the outdated dependency on `Tatter\Handlers`. I can't guarantee it will work the same as before though, lots has changed in two years.

**Comment by murugappanrm** [comment]
Hi,

Thank you for the update.

**Comment by eddgeo** [comment]
The plugin looks interesting, but I would like to know better how it works and how I can implement it and put it to use in a site that already has CI4. do you have screenshots?

**Comment by MGatner** [comment]
I wouldn't recommend using this. It is one of the last few libraries that I haven't reworked and is way overdue for an overhaul.

**Comment by murugappanrm** [comment]
I agree with @MGatner . The library may not integrate effectively with the latest versions of the host platforms.

**Comment by eddgeo** [comment]
Oh, too bad, the idea was very good, I will be attentive in case you dare to follow up on the update, I would love to use it in a future update that they do.

**Comment by MGatner** [comment]
Could you say more what you are looking for? Maybe describe the problem you are trying to solve?

Largely the reason why this got so little maintenance was I found it very rigid. It worked well for the original project I built it for, then I somewhat forced it into another; after that I found I preferred to write reports fresh because of the struggles of fitting this "scaffold". 

I do think a standardized way of approaching reports could be done, and maybe that is the future of this library, but I'd like more feedback before I try to solve a problem that doesn't really exist.

**Comment by murugappanrm** [comment]
hi @MGatner 

To give you an idea what i am looking for i would suggest you checkout Reportico.org. We are using that right now. Its fantastic but afraid that it may die soon or already dead as the developer has gone MIA.

**Comment by eddgeo** [comment]
> Could you say more what you are looking for? Maybe describe the problem you are trying to solve?
> 
> Largely the reason why this got so little maintenance was I found it very rigid. It worked well for the original project I built it for, then I somewhat forced it into another; after that I found I preferred to write reports fresh because of the struggles of fitting this "scaffold".
> 
> I do think a standardized way of approaching reports could be done, and maybe that is the future of this library, but I'd like more feedback before I try to solve a problem that doesn't really exist.

thanks for your reply

**Comment by eddgeo** [comment]
> hi @MGatner
> 
> To give you an idea what i am looking for i would suggest you checkout Reportico.org. We are using that right now. Its fantastic but afraid that it may die soon or already dead as the developer has gone MIA.

you have it for Codeigniter 4?, since I can't find how to implement it
